### PR TITLE
Fix design of 'sort' and 'filter' in 'People' and 'Companies' page

### PR DESCRIPTION
**Pull request by PepeGonzale**
#1485

### CODE REVIEW

File: front/src/modules/ui/view-bar/components/SortOrFilterChip.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ function SortOrFilterChip({
     <StyledChip isSort={isSort}>
       {Icon && (
         <StyledIcon>
-          <Icon />
+          <Icon size={14} />
```
Comment (lucasbordeau) [inline front/src/modules/ui/view-bar/components/SortOrFilterChip.tsx:70]: Can you use the theme for that ? I think it's `size.sm` ?

**Review by lucasbordeau — CHANGES_REQUESTED** [review]

File: front/src/modules/ui/view-bar/components/SortOrFilterChip.tsx (hunk 1 above)
Comment (lucasbordeau) [reply front/src/modules/ui/view-bar/components/SortOrFilterChip.tsx:70]: See the `useTheme` hook

Comment (PepeGonzale) [reply front/src/modules/ui/view-bar/components/SortOrFilterChip.tsx:70]: sure

Comment (PepeGonzale) [reply front/src/modules/ui/view-bar/components/SortOrFilterChip.tsx]: sure, fixed

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**